packages/gatsby-plugin-feed/package.json

```diff
@@ -23,7 +23,9 @@
   },
   "dependencies": {
     "babel-runtime": "^6.26.0",
-    "lodash.merge": "^4.6.0",
+    "deepmerge": "^2.0.1",
+    "lodash.defaultsdeep": "^4.6.0",
```

Comment:
I think we can get away with not including lodash. I know there's a gatsby plugin that integrates the babel plugin, but as a general rule, I'm trying to stay away from pulling in extras that I don't need. `deepmerge` is probably the most important, since most other config is dealt with through object rest spread.

Reply:
@KyleAMathews Do you have an official policy regarding lodash in gatsby plugins? Object rest spread seems to do most of what is needed here, and `deepmerge` fills in the gaps. But if it'd be better to rely on lodash if it's included in several plugins, I could see the case for using it.

Reply:
@nicholaswyoung Most recent commit drops both of the lodash packages. I replaced `defaultsDeep` with `deepmerge` and reversed the argument order, which _I think_ achieves the same result.